tests/s3/versioning/test_delete_object_tagging.py

```diff
@@ -348,3 +348,179 @@ def test_del_obj_tag_40436(self):
                                                     f"Expected: {put_tag} \n Actual: {get_tag}")
         LOGGER.info("ENDED: Test DELETE object tagging in a versioning enabled bucket.")
+
+    @pytest.mark.s3_ops
+    @pytest.mark.tags("TEST-40437")
+    def test_del_obj_tag_40437(self):
+        """
+        Test DELETE object tagging in a versioning suspended  bucket
+        """
+        LOGGER.info("STARTED: Test DELETE object tagging in a versioning suspended  bucket")
+
+        LOGGER.info("Step 1: PUT Bucket versioning with status as Enabled")
+        res = self.s3_ver_obj.put_bucket_versioning(bucket_name=self.bucket_name)
+        assert_utils.assert_true(res[0], res[1])
+        LOGGER.info("Step 1: PUT Bucket versioning with status as Enabled")
+        LOGGER.info("Step 2: Upload (creates version with versionId) of object - %s ", self.object_name)
+        s3_ver_tlib.upload_version(self.s3_test_obj, bucket_name=self.bucket_name,
+                                   file_path=self.file_path1, object_name=self.object_name,
+                                   versions_dict=self.versions)
+        latest_v = self.versions[self.object_name]["version_history"][-1]
+        LOGGER.info("Step 2: Uploaded (creates version with versionId) of object - %s ", self.object_name)
+        LOGGER.info("Step 3: Perform PUT Bucket versioning with status as Suspended")
+        res = self.s3_ver_obj.put_bucket_versioning(bucket_name=self.bucket_name, status="Suspended")
+        assert_utils.assert_true(res[0], res[1])
+        LOGGER.info("Step 3: Performed PUT Bucket versioning with status as Suspended")
+        LOGGER.info("Step 4: Perform PUT Object for object1 (creates version with null versionId)")
+        s3_ver_tlib.upload_version(self.s3_test_obj,
+                                   bucket_name=self.bucket_name,
+                                   file_path=self.file_path2,
+                                   object_name=self.object_name,
+                                   versions_dict=self.versions,
+                                   chk_null_version=True)
+
+        resp = s3_ver_tlib.put_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                              version_id="null",
+                                              s3_ver_test_obj=self.s3_ver_obj,
+                                              version_tag=self.ver_tag,
+                                              bucket_name=self.bucket_name,
+                                              object_name=self.object_name)
+        assert_utils.assert_true(resp[0], resp)
+        LOGGER.info("Step 4: Performed PUT Object for object1 (creates version with null versionId)")
+        ver_list = [latest_v, "null"]
+        LOGGER.info("Step 5: Perform PUT Object Tagging for object1 with a tag key-value pair for "
+                    "versionIds %s and null version %s", latest_v, "null")
```

Comment:
no need to do %s , "null" as you have already added that in statement

Reply:
ok, done